main.py

```diff
@@ -41,7 +41,7 @@ def main():
 		
 		for i in range(len(new_inputs)):
 			# get the sense of input (V/T/O/G/A)
-			sense = new_inputs[i][0]
+			sense, new_input, intensity = new_inputs[i].split(':')
```

Comment:
What is the new input format?
Are we taking input as T:hot:4?[1] or T_hot:4?[2] If [2] is followed than "sense, new_input, intensity = new_inputs[i].split(':')" this won't work.

Reply:
I think, there's one more commit after this change.
I've mentioned new input format in code as well.
T:hot:4 is the new input format